Makefile

```diff
@@ -95,6 +103,9 @@ build-updater: deps
 	@echo "+ $@"
 	go build -trimpath -o ./bin/updater ./cmd/updater
 
+.PHONY: image-build-args
+image-build-args:
+	@echo $(IMAGE_BUILD_ARGS)
```

Comment:
Rather than expose this here, _and_ duplicate the `docker build` command in the GHA yaml file, why not just call `make scanner-image...` in the GH action in the first place?
cc @BradLugo

Reply:
I'm going to merge as-is, as this repo is more-or-less in maintenance mode, so I think this is ok